einstein/einstein.lua

```diff
@@ -0,0 +1,91 @@
+--[[
+
+# [Salesforce Einstein Vision](https://metamind.readme.io/v2/docs) connector.
+
+### Authorization
+
+Authorization use [OAuth Authentication](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/intro_understanding_web_server_oauth_flow.htm).
+
+### Requirements
+
+  * [lua-multipart-post](https://github.com/catwell/lua-multipart-post) that is not installable via LuaRocks. This is required for `multipart/form-data` body encode and getting `boundary`.
+
+
+## Input parameters
+
+* 'access_token' - Bearer access token.
+* api_endpoint - API endpoint URI, e.g. `v2/vision/predict`.
+* method - optional request method, defauls to `GET`, could be `GET`, `POST`, `PUT`, `DELETE`.
+* post_data - optional JSON string used to post data.
+
+## Output parameters
+
+All connector output parameters could be used in PMIO Service Task output parameters as a value placeholders in curly braces.
+
+Example connector output parameters:
+
+```
+{
+  "probabilities": [
+    {
+      "label": "beach",
+      "probability": 0.9602110385894775
+    },
+    {
+      "label": "mountain",
+      "probability": 0.039788953959941864
+    }
+  ],
+  "object": "predictresponse"
+}
+```
+
+* response - JSON decoded structure received from API response.
+* code - HTTP response code.
+* headers - headers structure received from API response.
+]]--
+
+package.path = package.path .. ";./?.lua"
```

Comment:
you shouldn't change that path on the production code, since module would be installed via luarocks to the proper folder